src/main/cpp/t07_swap.cpp

```diff
@@ -14,10 +14,24 @@
 
 #include "t07_swap.h"
 #include <iostream>
-
+#include <vector>
 
 using namespace std;
 
 int t07_swap() {
-
+int a[100], n;
+cin >> n;
+for (int i=0; i<n; i++){
+cin >> a[i];
+}
+int elem;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'elem' can be reduced.](https://app.codacy.com/manual/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4719943)